app/assets/javascripts/discourse/app/instance-initializers/sidebar.js

```diff
@@ -0,0 +1,27 @@
+import I18n from "I18n";
+import { addSidebarPanel } from "discourse/lib/sidebar/custom-sections";
+
+export default {
+  initialize() {
+    addSidebarPanel((BaseCustomSidebarPanel) => {
+      const MainSidebarPanel = class extends BaseCustomSidebarPanel {
+        get key() {
+          return "main";
+        }
+        get switchButtonLabel() {
```

Comment:
```suggestion
        }

        get switchButtonLabel() {
```

small one for consistency